lib/socket.js

```diff
@@ -109,28 +109,29 @@ const socketFactory = (connect) => {
 //
 // see transportCommon.newConnectFn
 //
-const newConnectFn = (connect, transportName) =>
-  transportCommon.newConnectFn(
-    socketFactory(connect), Transport, transportName
-  );
+const newConnectFn = (connect, transportName) => transportCommon.newConnectFn(
+  socketFactory(connect), Transport, transportName
+);
 
 // Same as newConnectFn but will also perform inspect of specified
 // interfaces.
 //
 // see transportCommon.newConnectAndInspectFn
 //
-const newConnectAndInspectFn = (connect, transportName) =>
-  transportCommon.newConnectAndInspectFn(
-    socketFactory(connect), Transport, transportName
-  );
+const newConnectAndInspectFn =
+    (connect, transportName) => transportCommon.newConnectAndInspectFn(
+      socketFactory(connect), Transport, transportName
+    );
 
 // Same as newConnectFn but will instead perform reconnect on existing
 // connection.
 //
 // see transportCommon.newReconnectFn
 //
-const newReconnectFn = connect =>
-  transportCommon.newReconnectFn(socketFactory(connect), Transport);
+const newReconnectFn =
```

Comment:
You have enough space left to start function definition on the same line as you did on line 112 in this file.